lib/wrappers/output.ts

```diff
@@ -0,0 +1,61 @@
+import { PathLike } from "fs";
+import TransformNodeWrapper from "./transform-node";
+
+const fsExtra = require('fs-extra');
+const fs = require('fs');
+const logger = require('heimdalljs-logger')('broccoli:outputWrapper');
+const path = require('path');
+
+const WRITEOPERATION = new Set([
```

Comment:
I think we should be more conservative on which methods we allow. I don't think we should allow the nonSync variants as that may conflict with the lifecycle of other builds. What are your thoughts on this? cc @stefanpenner